lib/github.js

```diff
@@ -10,12 +10,38 @@ const graphqlWithAuth = graphql.defaults({
   }
 })
 
-module.exports.getPackageJson = async function getPackageJson (owner, repo) {
+module.exports.getDefaultBranch = async function getDefaultBranch (owner, repo) {
+  try {
+    const resp = await graphqlWithAuth({
+      query: `query ($owner: String!, $repo: String!) {
+        repository(name: $repo, owner: $owner) {
+          defaultBranchRef {
+            name
+        }
+      }
+    }`,
+      owner: owner,
+      repo: repo
+    })
+    if (resp && resp.repository && resp.repository.defaultBranchRef) {
+      return resp.repository.defaultBranchRef.name
+    }
+    throw Error(`Could not find GitHub repository at https://github.com/${owner}/${repo}`)
+  } catch (err) {
+    if (err.errors && err.errors[0].type === 'NOT_FOUND') {
+      throw Error(`Could not find GitHub repository at https://github.com/${owner}/${repo}`)
+    } else {
+      throw err
+    }
+  }
+}
+
+module.exports.getPackageJson = async function getPackageJson (owner, repo, defaultBranch) {
   try {
     const resp = await graphqlWithAuth({
       query: `query($owner: String!, $repo: String!) {
         repository(name: $repo, owner: $owner) {
-          object(expression: "master:package.json") {
+          object(expression: "${defaultBranch}:package.json") {
```

Comment:
```suggestion
          object(expression: "HEAD:package.json") {
```

This should read the `package.json` directly off the default branch, allowing us to skip that extra query?

Reply:
Maybe, let me look at this too. I have found the API from GIthub a bit hard to read and if this is better then we should do the simple thing.